pydra/engine/task.py

```diff
@@ -319,21 +342,28 @@ def command_args(self):
     def command_args(self, args: ty.Dict):
         self.inputs = attr.evolve(self.inputs, **args)
 
+    @property
+    def container_args(self):
+        """ there is no container args for the Shell Task."""
+        return []
+
     @property
     def cmdline(self):
         """Get the actual command line that will be submitted."""
         orig_inputs = attr.asdict(self.inputs)
         modified_inputs = template_update(self.inputs)
         if modified_inputs is not None:
             self.inputs = attr.evolve(self.inputs, **modified_inputs)
-        cmdline = " ".join(self.command_args)
+        cmdline = " ".join(self.container_args + self.command_args)
```

Comment:
should this and the container cmdline be reverted.

Reply:
not sure if i follow, looks good to me